docker-compose.override.yml

```diff
@@ -25,14 +26,22 @@ services:
     fpm:
         volumes:
             - ./:/srv/journal
+            - ./.docker/php.ini:/usr/local/etc/php/conf.d/x-dev.ini
             - /srv/journal/build
             - /srv/journal/var
+            - critical_css:/srv/journal/build/critical-css
             - vendor:/srv/journal/vendor
     web:
         volumes:
             - assets:/srv/journal/web
+    critical_css:
+        command: node_modules/.bin/gulp critical-css:generate
```

Comment:
convenient, although as in https://github.com/elifesciences/journal/pull/1024/files#r202701264 makes `dev` and `ci` different

Reply:
Probably can make the `CMD` and carry on overriding in CI.